src/orangeClockFunctions/displayBlockMoscowFees.py

```diff
@@ -1,284 +1,287 @@
+
+
+def getPrice(currency): # change USD to EUR for price in euro
+    gc.collect()
+    data = urequests.get("https://mempool.space/api/v1/prices")
+    price = data.json()[currency]
+    data.close()
+    return price
+
+
+def getMoscowTime():
+    moscowTime = str(int(100000000 / float(getPrice("USD"))))
+    return moscowTime
+
+
+def getLastBlock():
+    gc.collect()
+    data = urequests.get("https://mempool.space/api/blocks/tip/height")
+    block = data.text
+    data.close()
+    return block
+
+
+def getMempoolFees():
+    gc.collect()
+    data = urequests.get("https://mempool.space/api/v1/fees/recommended")
+    jsonData = data.json()
+    data.close()
+    return jsonData
+
+
+def getMempoolFeesString():
+    mempoolFees = getMempoolFees()
+    mempoolFeesString = (
+        "L:"
+        + str(mempoolFees["hourFee"])
+        + " M:"
+        + str(mempoolFees["halfHourFee"])
+        + " H:"
+        + str(mempoolFees["fastestFee"])
+    )
+    return mempoolFeesString
+
+
+def displayInit():
+    refresh(ssd, True)
+    ssd.wait_until_ready()
+    time.sleep(5)
+    ssd._full = False
+    ssd.wait_until_ready()
+    refresh(ssd, True)
+    ssd.wait_until_ready()
+    ssd.sleep()  # deep sleep
+    time.sleep(5)
+
+
+def debugConsoleOutput(id):
+    print("===============debug id= " + id + "===============")
+    print("memory use: ", gc.mem_alloc() / 1024, "KiB")
+    print("memory free: ", gc.mem_free() / 1024, "KiB")
+    print("===============end debug===============")
+
+
+def main():
+    gc.enable()
+    global wifi
+    global secretsSSID
+    global secretsPASSWORD
+    debugConsoleOutput("1")
+    issue = False
+    blockHeight = ""
+    textRow2 = ""
+    mempoolFees = ""
+    i = 1
+    connectWIFI()
+    displayInit()
+    while True:
+        debugConsoleOutput("2")
+        if issue:
+            issue = False
+        if i > 72:
+            i = 1
+            refresh(ssd, True)  # awake from deep sleep
+            time.sleep(5)
+            ssd._full = True
+            ssd.wait_until_ready()
+            refresh(ssd, True)
+            ssd.wait_until_ready()
+            time.sleep(20)
+            ssd._full = False
+            ssd.wait_until_ready()
+            refresh(ssd, True)
+            time.sleep(5)
+        try:
+            symbolRow1 = "A"
+            blockHeight = getLastBlock()
+        except Exception as err:
+            blockHeight = "connection error"
+            symbolRow1 = ""
+            print("Block: Handling run-time error:", err)
+            debugConsoleOutput("3")
+            issue = True
+        try:
+            if dispVersion == "mt":
+                symbolRow2 = ""
+                textRow2 = getMoscowTime()
+            elif dispVersion == "mts2":
+                symbolRow2 = "H"
+                textRow2 = getMoscowTime()
+            elif dispVersion == "fp1":
+                symbolRow2 = "J"
+                textRow2 = str(getPrice("USD"))

```

Comment:
I think it would be better to add another elif for the new symbol.
At the moment your PR removes the $ symbol completely as an option